build/teamcity/cockroach/nightlies/pebble_nightly_metamorphic_crossversion_impl.sh

```diff
@@ -31,6 +31,7 @@ test_args="--test_arg=-test.v $test_args"
 
 $BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci -- \
                                       test @com_github_cockroachdb_pebble//internal/metamorphic/crossversion:crossversion_test \
+                                      --test_env TC_SERVER_URL=$TC_SERVER_URL \
```

Comment:
Nit: You can use `--test_env TC_SERVER_URL` if you want to inherit the value of `TC_SERVER_URL` from the environment.